app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', counter_cache: :posts_counter
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  def update_posts_count
+    author.update(posts_counter: author.posts.count)
+  end
+
+  def recent_comments
+    comments.last(5)
+  end
+end
```

Comment:
OPTIONAL
- You can also consider using the helper method `after_save` to ensure that with every update related to this specific model `Post` then invoke the `update_posts_count` method immediately. 👍

Reply:
This change has been done well